doc/example_code/how_to_examples/index.rst

```diff
@@ -403,6 +404,8 @@ Backgrounds
 
    :ref:`background_parallax`
 
+.. _examples_cameras:
+
```

Comment:
Should we be consistent in naming? 'sprite_examples' and 'camera_examples'?

Reply:
My intent with heading names was to reflect heading hierarchy from left to right, like a file path. To my knowledge, sphinx doesn't support this sort of nesting outside of toctree declarations, but I might be wrong.

I'd like to switch the other instances to follow this pattern, but I'm not sure it's appropriate to do so in this PR.

Your thoughts?

Reply:
I think it is inconsistent to have examples_sprites and camera_examples. I get the header-thing but this really sticks out to me for some reason.

Reply:
I think you're misunderstanding: I am proposing we rename everything to `examples_subtype`